### PR TITLE
Support PagerDuty Alarm Hook

### LINKED ISSUE #7572: Support PagerDuty immigration for alarm
PagerDuty(https://www.pagerduty.com/) is widely used for on-call globally. We are welcome the contributions to support it as alarm notification channel.

### PR DESCRIPTION
**Pull request by chen-ni**
### <Feature description>
- [x] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [x] If it's UI related, attach the screenshots below.

- [x] Closes #7572.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/changelog/docs/en/changes/changes.md).

# Example of triggered incidents on PagerDuty
Screen Shot 2022-06-18 at 14 09 25

### CODE REVIEW

**Comment by chen-ni** [comment]
@wu-sheng The changelog link in the PR template is no longer valid. 

However, I found this one: https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md. Should I update it?

**Comment by wu-sheng** [comment]
I can see template is updated, what do you mean?

https://github.com/apache/skywalking/blob/master/.github/PULL_REQUEST_TEMPLATE#L33

**Comment by wu-sheng** [comment]
And you should make CI and e2e passed locally. 
Please attach a screenshot to verify how this works

**Comment by chen-ni** [comment]
@wu-sheng OK, I'll fix CI / E2E first.

**Comment by chen-ni** [comment]
@wu-sheng Just fixed a checkStyle error. But I don't know how to run all the CI / E2E workflows locally, could you please show me how?

**Comment by wu-sheng** [comment]
This is your first PR to this repository, so I(or another committer) have to approve for every commit you made. Those tasks would run automatically once this first PR gets merged.

**Comment by wu-sheng** [comment]
image

Where are the values of these two fields from?

**Comment by chen-ni** [comment]
@wu-sheng The Title field is just the alarm message we're sending. In this case, the text template is set to `"Apache SkyWalking Alarm: \n %s."`, and the `%s` is replaced with the alarm message `anotherAlarmMessage.setAlarmMessage("anotherAlarmMessage with [DefaultScopeDefine.Endpoint]");
`, so the formatted message becomes the one you see.

The Service field has nothing to do with us. It's the name that the PagerDuty user gives to their service. Basically:

- The user creates a service on PagerDuty and gives it a name (like in this case, "test service");
- The user creates an "integration" for the service and copies the integration key;
- The user configures the integration key(s) in our YAML file. So when an alarm is triggered, the message is sent to the service(s) that corresponds to the integration key(s).

File: oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml
Comment (wu-sheng) [inline oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml:138]: This file should be updated too, https://github.com/apache/skywalking/blob/f8203778efad77e1d1fb14f10a0000014b1a8517/oap-server/server-starter/src/main/resources/alarm-settings.yml

**Comment by wu-sheng** [comment]
> @wu-sheng The changelog link in the PR template is no longer valid.
> 
> However, I found this one: https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md. Should I update it?

Oh, I noticed what is breaking here. If you want, feel free to fix this through another PR. Or I could do this fix too. Wait for your reply.

**Comment by wu-sheng** [comment]
And don't forget to update changelog for your own PR. :)

File: oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml
Comment (chen-ni) [reply oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml:138]: @wu-sheng Thanks, fixed in https://github.com/apache/skywalking/pull/9247/commits/5cbb7388f94ed85261a2468f2940d2153e8e45cd :)

**Comment by chen-ni** [comment]
@wu-sheng Yeah, I'd like to create another PR to fix the changelog link in the PR template :)

And just updated the changelog for this PR here: https://github.com/apache/skywalking/pull/9247/commits/1a48697e3ea32acefc22daba68780d4799f8a286

**Review by wu-sheng — APPROVED** [review]
LGTM. @kezhenxu94 PTAL

**Review by kezhenxu94 — APPROVED** [review]
lgtm

**Pull request merged**
